### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ttsmousavi** [thingsboard/thingsboard-gateway#375]
**Describe the bug**
From dashBoard, the user cannot update (set value) attributes that is not a string.
DASHBOARD -> Create "Input widgets" -> "Update shared Integer attribute"

GW throw an exception 

**Connector name (If bug in the some connector):**
MQTT Connector

**Error traceback (If available):**
```
replace() argument 2 must be str, not bool
/usr/lib/python3/dist-packages/thingsboard_gateway/connectors/mqtt/mqtt_connector.py Line:539 and 532

""2020-08-14 15:35:56" - ERROR - [tb_gateway_service.py] - tb_gateway_service - 591 - replace() argument 2 must be str, not int"
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/thingsboard_gateway/gateway/tb_gateway_service.py", line 589, in _attribute_update_callback
    self.__connected_devices[content["device"]]["connector"].on_attributes_update(content)
  File "/usr/lib/python3/dist-packages/thingsboard_gateway/connectors/mqtt/mqtt_connector.py", line 532, in on_attributes_update
    .replace("${attributeValue}", content["data"][attribute_key])
TypeError: replace() argument 2 must be str, not int
""2020-08-14 15:35:56" - ERROR - [tb_gateway_service.py] - tb_gateway_service - 591 - replace() argument 2 must be str, not int"
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/thingsboard_gateway/gateway/tb_gateway_service.py", line 589, in _attribute_update_callback
    self.__connected_devices[content["device"]]["connector"].on_attributes_update(content)
  File "/usr/lib/python3/dist-packages/thingsboard_gateway/connectors/mqtt/mqtt_connector.py", line 532, in on_attributes_update
    .replace("${attributeValue}", content["data"][attribute_key])
TypeError: replace() argument 2 must be str, not int



Solution: replacing  <content["data"][attribute_key]> to <str(content["data"][attribute_key])>
```

**Versions (please complete the following information):**
 - OS: [Ubuntu 18.04]
 - Thingsboard IoT Gateway version [2.5.0]
 - Python version[2.7.17]

**Comment by ExPS-Pisko** [comment]
Hi ttsmousavi, I have experienced the same problem. The thing is that the INT type in Thingsboard is a LongINT (64-bit). Check in your OPC server (for example using the UA Expert or some tool from Matrikon) the datatype of the variable you want to write into. Once I changed from INT to this LongINT in my PLC, the write is OK without failure.

The same with other data-types (Real, String...)

Best regards Michal
